frontend/src/api/authRequest.test.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable max-nested-callbacks */
+import {AuthApiClient} from './authRequest'
+// import '@types/jest'
+
+describe('AuthApiClient', () => {
+  test('make a login request', async () => {
+    const expected = [
+      { email: 'test@example.com', password: '123456' }
+    ]
+
+    jest.spyOn(window, 'fetch').mockImplementation(() => {
```

Comment:
I think it will be more idomatic if we will mock here the `HttpClient` instead of fetch. So we're rely on `HttpClient` *interface*, not but it's *implementation*. Just in case, if we want to replace `fetch` inside `HttpClient` with some another method - then we don't need to rewrite tests. But if we leave this test as is - any internal changes of `HttpClient` will impact on this test.
*General idea*: you need to write unit test in way that allow you to no bother about other dependencies of tested module.

Reply:
I changed HttpClient by its mock and now it just checkes that authRequest is working. Tests are passed.